packages/mobile/src/app/saga.ts

```diff
@@ -114,6 +120,9 @@ export function* appRemoteFeatureFlagSaga() {
       const flags: RemoteFeatureFlags = yield take(remoteFeatureFlagChannel)
       Logger.info(TAG, 'Updated feature flags', JSON.stringify(flags))
       yield put(updateFeatureFlags(flags))
+      const accountAddress: string = yield select(accountAddressSelector)
+      const rewardsEnabled = accountAddress < flags.rewardsABTestThreshold
+      yield put(setRewardsEnabled(rewardsEnabled))
```

Comment:
Instead of doing this, how about just saving the `rewardsABTestThreshold` in the redux store and doing this calculation in a selector?

Not sure if this would work when creating an account otherwise since this would be run before there's an `accountAddress` and it's never updated unless something changes in the Firebase db.